.github/workflows/publish.yml

```diff
@@ -1,71 +1,387 @@
+        run: ./gradlew publishMacosX64PublicationToDeployRepository
+
+      - name: Publish macos Arm64
+        working-directory: arrow-libs
+        run: ./gradlew publishMacosArm64PublicationToDeployRepository
+
+      - name: Publish iosX64
+        working-directory: arrow-libs
+        run: ./gradlew publishIosX64PublicationToDeployRepository
+
+      - name: Publish iosArm64
+        working-directory: arrow-libs
+        run: ./gradlew publishIosArm64PublicationToDeployRepository
 
+      - name: Publish iosArm32
+        working-directory: arrow-libs
+        run: ./gradlew publishIosArm32PublicationToDeployRepository
+
+  publish_tvos_and_watchos:
+    runs-on: macos-latest
+    timeout-minutes: 20
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Restore Gradle cache
+        id: cache
+        uses: actions/cache@v2.1.6
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+            ~/.konan
+          key: tv-step-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            tv-step-gradle-
+
+      - name: Publish tvos Arm64
+        working-directory: arrow-libs
+        run: ./gradlew publishTvosArm64PublicationToDeployRepository
+
+      - name: Publish tvos X64
+        working-directory: arrow-libs
+        run: ./gradlew publishTvosX64PublicationToDeployRepository
+
+      - name: Publish tvosSimulatorArm64
+        working-directory: arrow-libs
+        run: ./gradlew publishTvosSimulatorArm64PublicationToDeployRepository
+
+      - name: Publish watchosX64
+        working-directory: arrow-libs
+        run: ./gradlew publishWatchosX64PublicationToDeployRepository
+
+      - name: Publish watchosX86
+        working-directory: arrow-libs
+        run: ./gradlew publishWatchosX86PublicationToDeployRepository
+
+      - name: Publish watchosArm32
+        working-directory: arrow-libs
+        run: ./gradlew publishWatchosArm32PublicationToDeployRepository
+
+      - name: Publish watchosArm64
+        working-directory: arrow-libs
+        run: ./gradlew publishWatchosArm64PublicationToDeployRepository
+
+  publish_windows:
+    runs-on: windows-latest
+    timeout-minutes: 20
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Restore Gradle cache
+        id: cache
+        uses: actions/cache@v2.1.6
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+            ~/.konan
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Publish MingwX64
+        working-directory: arrow-libs
+        run: ./gradlew publishMingwX64PublicationToDeployRepository
+  publish-arrow-stack:
     env:
       BASEDIR: ${{github.workspace}}/arrow-libs
```

Comment:
Is this used? We changed to `working-directory: arrow-stack` in the `run` step.

Reply:
the env variable is not affecting this step `working-directory: arrow-stack`